From caf1c2836e333accce23175cc139465b4eef2a02 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Mon, 09 Jun 2025 14:37:06 +0300
Subject: [PATCH] [SW-231789] g2e: fix incorrect cherry-pick

During cherry-pick from master_next to v1.21.2_next,
a merge conflict was solved incorrectly and missed by CR.
The fuse's fields are not updated as a result.

Change-Id: Ic963e800f9b090c1d604d76d23b33a819e4916db
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index b8ad2e3..d1c0df8 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -377,7 +377,6 @@
 			/* HL-288 is China only card so BIS E is set by default,
 			 * unless fake security is enabled
 			 */
-		} else if (is_card_type_pcie()) {
 			fuse->bis_compliant = 1;
 			fuse->bis_e_compliant = 1;
 		}
